lib/modules/manager/gleam/__fixtures__/gleam.dev_deps_only.toml

```diff
@@ -0,0 +1,6 @@
+name = "test_gleam_toml"
```

Comment:
inline those small fixtures to the spec test file and use `codeBlock` string template helper